cmd/functions.go

```diff
@@ -89,6 +90,7 @@ func init() {
 	functionsDeleteCmd.Flags().StringVar(&projectRef, "project-ref", "", "Project ref of the Supabase project.")
 	functionsDeployCmd.Flags().BoolVar(&noVerifyJWT, "no-verify-jwt", false, "Disable JWT verification for the Function.")
 	functionsDeployCmd.Flags().StringVar(&projectRef, "project-ref", "", "Project ref of the Supabase project.")
+	functionsDeployCmd.Flags().BoolVar(&useLegacyBundle, "legacy-bundle", false, "Use legacy bundling mechanism.")
```

Comment:
Do you think it's worth defaulting to `true` initially and flip it may be next week? I assume we haven't tested on windows?

Reply:
Yup, good call on testing on Windows. Can we do a pre-release for testing? I have set the flag to true in 6603bde